cvat/apps/tf_annotation/views.py

```diff
@@ -203,7 +203,7 @@ def create_thread(tid, labels_mapping, user):
         result = convert_to_cvat_format(result)
         serializer = LabeledDataSerializer(data = result)
         if serializer.is_valid(raise_exception=True):
-            put_task_data(tid, user, result)
+            put_task_data(tid, result)
```

Comment:
@zhiltsov-max , could you please explain why we don't need `user` here anymore? Not all users should be able to run put_task_data function.

Reply:
This parameter was never used in any of these functions, or in Task- and JobAnnotations classes.